### PR TITLE
Ability to add conversation TTL to redis client

### PR DESCRIPTION
**Pull request by cabezayunke**
When adding a TTL with the setConversationTTL method, it will redis.expire with that value after the redis.set command.

### CODE REVIEW

File: __tests__/storage.spec.ts [hunk 1]
```diff
@@ -49,6 +61,7 @@ describe("RedisStorage", () => {
           done()
         })
       })
+
```
Comment (bilby91) [inline __tests__/storage.spec.ts]: Remove this space please

File: __tests__/storage.spec.ts [hunk 2]
```diff
@@ -72,5 +85,52 @@ describe("RedisStorage", () => {
...
+    describe("when there is a TTL set", () => {
+        let newData
+
+        beforeEach(() => {
+            newData =  { userData: { a: "b" }, privateConversationData: { c: "d" }, conversationData: { e: "f" } }
+        })
+
+        it("sets the TTL correctly", (done) => {
+            storage.setConversationTTL(100)
```
Comment (bilby91) [inline __tests__/storage.spec.ts]: Please, be consistent with 2 spaces for indentation.

File: __tests__/storage.spec.ts [hunk 3]
```diff
@@ -72,5 +85,52 @@ describe("RedisStorage", () => {
         })
       })
     })
+
+    describe("when there is a TTL set", () => {
```
Comment (bilby91) [inline __tests__/storage.spec.ts]: I think we are missing the most important test case here. I would add a new test that checks the following.

Set a ttl of 100, saveData, wait 200, check that data is no longer present.

Make sense ?

File: package.json [hunk 4]
```diff
@@ -45,7 +45,7 @@
     "redis"
   ],
   "dependencies": {
-    "botbuilder": "^3.8.0",
-    "redis": "^2.6.0"
+    "botbuilder": "3.14.0",
```
Comment (bilby91) [inline package.json]: I prefer to keep the old constraints, people with older versions might want to use this dependency :)

**Comment by bilby91** [comment]
@cabezayunke Awesome work! One more thing, can you squash your changes into one commit please ? 

Thanks again!

File: __tests__/storage.spec.ts (hunk 2 above)
Comment (cabezayunke) [reply __tests__/storage.spec.ts]: I will re-indent everything, I have my IDE configured for 4 spaces indentation.

**Comment by bilby91** [comment]
Duplicate #30

**Pull request closed without merging**